metpy/gridding/tests/test_gridding_functions.py

```diff
@@ -126,3 +126,32 @@ def test_interpolate(method, test_coords):
         truth = np.load(fobj)['img']
 
     assert_array_almost_equal(truth, img)
+
+
+@pytest.mark.parametrize('method', interp_methods)
+def test_interpolate_manualgrid(method, test_coords):
+    r"""Test interpolate function using user-defined boundaries."""
+    xp, yp = test_coords
+
+    xp *= 10
+    yp *= 10
+
+    z = np.array([0.064, 4.489, 6.241, 0.1, 2.704, 2.809, 9.604, 1.156,
+                  0.225, 3.364])
+
+    extra_kw = {}
+    if method == 'cressman':
+        extra_kw['search_radius'] = 200
+        extra_kw['minimum_neighbors'] = 1
+    elif method == 'barnes':
+        extra_kw['search_radius'] = 400
+        extra_kw['minimum_neighbors'] = 1
+        extra_kw['gamma'] = 1
+    testCoords = {'west': 80.0, 'south': 140.0, 'east': 980.0, 'north': 980.0}
+    _, _, img = interpolate(xp, yp, z, hres=10, interp_type=method, **extra_kw,
+                            boundary_coords=testCoords)
```

Comment:
You can't have this argument like this after the `**kwargs`.

Reply:
Oops... I'll fix that